delta/delta.schema.json

```diff
@@ -2096,13 +2093,12 @@
         "parent",
         "reference",
         "index",
-        "target",
         "newResolveInfo",
         "commandId"
       ],
       "additionalProperties": false,
-      "minProperties": 7,
-      "maxProperties": 8
+      "minProperties": 6,
```

Comment:
Is it useful to constrain the number of properties like this, beyond specifying the `requiredProperties`?

Reply:
As I understand JsonSchema, it only limits the possible space of values. Without that, we could add five times e.g. `newResolveInfo` (as JSON doesn't require unique "keys").